packages/grafana-ui/src/components/Table/utils.ts

```diff
@@ -477,3 +506,18 @@ export const defaultRowNumberColumnFieldData: Omit<Field, 'values'> = {
     },
   },
 };
+
```

Comment:
The comments are great! Let's add a TSDoc comment for the function here as well 😄